Django Database.csv

```diff
@@ -0,0 +1,5 @@
+id,user_email,task,due_by,priority,is_urgent
```

Comment:
[nit] I would name this file something like `tasks.csv` and put it in a folder like `./tasks/data/`. Might be a good idea to stay away from file names with spaces in it

Reply:
you do make a point, I just went with this process thinking how to solve it xD thank you